web/src/util/normalize-item.js

```diff
@@ -3,6 +3,7 @@ export default function normalizeItem(item) {
     {},
     {
       itemID: item.sale_item_id,
+      resourceID: item.sale_item_resource_id,
```

Comment:
Doe we need to remove any of the other IDs? 
Is `resourceID` the same as `itemID` (Which is the UID of the NFT) ?